### PR TITLE
Azure Site Recovery - Adding new optional parameters to support additional networking resources

### PR DESCRIPTION
**Pull request by punit-bhatt**
## Description

[Azure/azure-powershell-cmdlet-review-pr#362](https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/362)

## Checklist

- [x] I have read the [_Submitting Changes_](../CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [x] the markdown help files have been regenerated using the commands listed [here](../documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

**Comment by wyunchi-ms** [comment]
Hi @punit1396

File: tools/Common.Netcore.Dependencies.targets [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <ItemGroup > 
-    <PackageReference Include="Microsoft.Rest.ClientRuntime" Version="2.3.19"/>
```
[the remark this answers is not in the document]
Comment (punit1396) [reply tools/Common.Netcore.Dependencies.targets]: Powershell MSI generation fails with the error - RecoveryServices\RecoveryServices.SiteRecovery\RecoveryServices.SiteRecovery.csproj : error NU1605: Detected package downgrade: Microsoft.Rest.ClientRuntime from 2.3.20 to 2.3.19.

The new pkg that RecoveryServices.SiteRecoveyr uses requires 2.3.20.

**Comment by punit1396** [comment]
Hi @wyunchi-ms . I have removed the nupkg from the localFeed.

**Comment by wyunchi-ms** [comment]
Hi @punit1396 
Could you please fix the linux CI problem? You can pick Mark's fix [here](https://github.com/Azure/azure-powershell/pull/10209/commits)
Thanks

File: tools/Common.Netcore.Dependencies.targets (hunk 1 above)
[the remark this answers is not in the document]
Comment (wyunchi-ms) [reply tools/Common.Netcore.Dependencies.targets]: Is MSI generation nessary?

[the remark this answers is not in the document]
Comment (punit1396) [reply tools/Common.Netcore.Dependencies.targets]: We needed that urgently for a few customers. And have one now. 
If this is brought back to 2.3.19, wouldn't that be an issue during the actual release?

**Pull request merged**